website/docs/versioning.md

```diff
@@ -3,15 +3,11 @@ id: versioning
 title: Versioning
 ---
 
-You can use the version script to cut a new documentation version based on the latest content in the `docs` directory. That specific set of documentation will then be preserved and accessible even as the documentation in the `docs` directory changes moving forward.
+You can use the version script to create a new documentation version based on the latest content in the `docs` directory. That specific set of documentation will then be preserved and accessible even as the documentation in the `docs` directory changes moving forward.
 
-:::caution
+:::caution Think about it before starting to version your documentation - it can become difficult for contributors to help improve it! :::
```

Comment:
This is incorrect - we need `:::caution` to be on its own line. See https://deploy-preview-2606--docusaurus-2.netlify.com/docs/next/versioning for why it's incorrect.

Reply:
Whoops - that was unintentional

Reply:
This was a common problem - Prettier formats the code and combines the line after `:::caution` . So we intentionally left an empty line after `:::caution`. All the admonitions in the docs are written like this :)